File: src/ch/pajama/boredomkiller/db/model/Statistic.java
package ch.pajama.boredomkiller.db.model;

public class Statistic {
	private int timesPlayed = 0;
	private int totalPlaytime = 0;
	
	public Statistic(int timesPlayed, int totalPlaytime){
		this.timesPlayed = timesPlayed;
		this.totalPlaytime = totalPlaytime;
	}

	public int getTimesPlayed() {
		return timesPlayed;
	}

	public void setTimesPlayed(int timesPlayed) {
		this.timesPlayed = timesPlayed;
	}

	public int getTotalPlaytime() {
		return totalPlaytime;
	}

	public void setTotalPlaytime(int totalPlaytime) {
		this.totalPlaytime = totalPlaytime;
	}
}

File: src/webadmin/application/models/statistic.php
<?php

/**
 * Model for the tbl_statistics table
 *
 * @author patrick
 */
class Statistic extends BK_Model
{

    /**
     * Table name
     */
    const dbTable = 'tbl_statistics';
    
    /**
     * Statistic id
     * @var integer
     */
    public $id;
    
    /**
     * Number of plays
     * @var integer
     */
    public $times_played;
    
    /**
     * Playtime timestamp
     * @var integer
     */
    public $total_playtime;
    
    /**
     * Class constructor
     */
    public function __construct()
    {
        parent::__construct();
    }
    
    /**
     * Id getter
     * @return integer
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Id setter
     * @param integer $id
     * @return boolean|\Statistic
     */
    public function setId($id)
    {
        $id = intval($id);
        if ($id < 0) {
            return FALSE;
        }
        $this->id = $id;
        return $this;
    }

    /**
     * Times played getter
     * @return integer
     */
    public function getTimesPlayed()
    {
        return $this->times_played;
    }

    /**
     * Times played setter
     * @param integer $times_played
     * @return \Statistic
     */
    public function setTimesPlayed($times_played)
    {
        $this->times_played = $times_played;
        return $this;
    }

    /**
     * Playtime getter
     * @return integer
     */
    public function getTotalPlaytime()
    {
        return $this->total_playtime;
    }

    /**
     * Playtime setter
     * @param integer $total_playtime
     * @return \Statistic
     */
    public function setTotalPlaytime($total_playtime)
    {
        $this->total_playtime = $total_playtime;
        return $this;
    }

}
